docs/2023/cQube/cQubeChat/updates/2023-07-07.md

```diff
@@ -4,13 +4,22 @@ author: Suyash Gautam
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [x] Understanding cQube, implemetation and database design
+- [x] Learning langchain and streamlit
+- [x] Cleaning the cQube database dump
+- [x] POC on different ways to create an interface to intereact with ChatGPT
+- [x] Coming up a with a high level diagram for cQubeChat
 
 ## Screenshots / Videos 
 
+![High Level Diagram](./Text2SQL%20and%20cQubeChat-cQubeChat%20(1).jpg)
+
 ## Contributions
 
+- https://github.com/Sunbird-cQube/cQubeChat/issues/10
+- https://github.com/Sunbird-cQube/cQubeChat/issues/7
+
 ## Learnings
+
+SQLDatabaseChain is good for cases where we have we have small database and interactive table names.
+Streamlit provides ways to create chart which will become useful later on to show charts for the data.
```

Comment:
nit:
```bash

- SQLDatabaseChain is good for cases where we have we have small database and interactive table names.

- Streamlit provides ways to create chart which will become useful later on to show charts for the data.
```

Reply:
I have fixed the issue in the latest commit

Reply:
Looks good, but generally you should add the reference to your commit in your comment.
It's just a good practice.